src/pages/index.astro

```diff
@@ -113,22 +120,16 @@ const cardObjects = [weatherCard, calculatorCard, realEstateCard, forestCard];
         <div class="mt-24">
           <ul class="grid md:grid-cols-2 gap-8 md:gap-16">
             {
-              cardObjects.map((card) => {
-                return (
-                  <li class="flex justify-stretch">
-                    <CardProject
-                      img={{
-                        classProp: "rounded-3xl w-64 h-64",
-                        src: card.imgSrc,
-                        alt: card.imgAlt,
-                      }},
-                      title = {card.title},
-                      description= {card.description},
-                      skills={card.skills}
-                    />
-                  </li>
-                );
-              })
+              cards.map((card) => (
+                <li class="flex justify-stretch">
+                  <CardProject
+                    img={card.img}
+                    title={card.title}
+                    description={card.description}
+                    skills={card.skills}
+                  />
```

Comment:
Much better, one last request to move on. All these props come from the same source - let's adjust the Card props to accept just `card` containing all existing card data

Reply:
Changed